Source/Scene/ShadowMap.js

```diff
@@ -135,6 +136,7 @@ define([
         //>>includeEnd('debug');
 
         this.enabled = false;
+        this._softShadows = defaultValue(options.softShadows, false);
```

Comment:
Can we expose this as a settable option?  If it is too painful to make it dynamic, we should at least be able to set it through Scene/Widget/Viewer at construction time.